civis/io/_tables.py

```diff
@@ -224,7 +258,12 @@ def f(x):
                                .format(script_id))
     url = outputs[0]["path"]
     if use_pandas:
-        data = pd.read_csv(url, **kwargs)
+        sortkeys, py_types, date_cols = _redshift_to_py(client, table_id)
+        # Not sure about the best way to handle null fields
+        data = pd.read_csv(url, parse_dates=date_cols,
+                                index_col=sortkeys,
+                                converters=py_types,
+                                engine='python')
```

Comment:
Can you explain this particular invocation of `pd.read_csv`? Also, we should not be setting the index of the output DF here. The convention used here IIUIC is that the DF row index is always integers and any dist or sort keys in redshift are left as normal columns. Finally, I don't think we need to revert to the python engine here, but I could be wrong.

Reply:
I'll drop the sort keys. The python engine is much more robust and sometimes you get weird errors trying to make type conversions with the c engine, furthermore the performance difference seems negligible. When you say invocation what do you mean, the keyword args? The original was `pd.read_csv(url)`

Reply:
Yeah I just meant the overall function signature and the choice of keywords. Have you tested performance with very big tables? I think this might be where that matters but I am not sure.

Reply:
I have a table that's over 1.5GB and I can't say I noticed any difference switching between the two but it did crash when there was an integer column that had some funny formatting when using the C engine. See preformance comparison: [here](https://stackoverflow.com/questions/29678154/why-is-numpy-pandas-parsing-of-a-csv-file-with-long-lines-so-slow)